README.md

```diff
@@ -5,7 +5,7 @@
 
 This is a productivity tool which draws some inspiration from workflowy and vim.
 
-[Try it out!](https://vimflowy.bitballoon.com)
+[![Bitballoon logo](/docs/media/bitballoon-logo.png) Try it out!](https://vimflowy.bitballoon.com) or [![Docker logo](/docs/media/docker-logo.png) Deploy to docker](https://hub.docker.com/r/vimflowy/vimflowy/)
```

Comment:
should it be "deploy with docker"?

Reply:
I think it could be either, as docker is also a container runtime (more specifically runc is the runtime, but people just say docker). I'll change it to *with* as it reads a little nicer.